std-bits/base/src/main/java/org/enso/base/net/URIHelpers.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.base.net;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import org.apache.http.client.utils.URIBuilder;
```

Comment:
I believe there is similar [builder in the JDK](https://docs.oracle.com/en/java/javase/11/docs/api/java.net.http/java/net/http/HttpClient.Builder.html) as well.

Reply:
How is it similar?

This is a HttpClient.Builder, I cannot see any capabilities for building an URI with it.